src/couch/src/couch_httpd_vhost.erl

```diff
@@ -394,25 +390,23 @@ handle_config_terminate(_Server, _Reason, _State) ->
 
 load_conf() ->
     %% get vhost globals
-    VHostGlobals = re:split(config:get("httpd",
-            "vhost_global_handlers",""), "\\s*,\\s*",[{return, list}]),
+    VHostGlobals = re:split("_utils, _uuids, _session, _users", "\\s*,\\s*",
```

Comment:
Did you consider hardcoding it?:
```
VHostGlobals = ["_utils","_uuids","_session","_users"]
```

Reply:
good call, might have to keep it dynamic to allow `couch.app` configurability as per 
https://github.com/apache/couchdb/pull/1602#issuecomment-42200131 but good call2